suite-common/fiat-services/src/coingecko.ts

```diff
@@ -171,3 +172,11 @@ export const fetchLastWeekRates = async (
         ts: new Date().getTime(),
     };
 };
+
+export const coingeckoService = {
```

Comment:
I would suggest simply use `import * as coingeckoService from './coingecko'`, this wrapping inside of object is useful for actions only in wallet-core because there is multiple other files that will import these actions and people then can choose different names when doing `import * as neco`, but here it's imported only in one place and it's not even exported from package

same for blockbookServices